src/soho/column/soho-column.d.ts

```diff
@@ -75,13 +75,17 @@ interface SohoColumn {
 type SohoColumnSelected = SohoColumnFieldSelected | SohoColumnGroupSelected;
 
 interface SohoColumnFieldSelected {
-  fieldName: string;
-  fieldValue: any;
+  // USe either index or fieldName and fieldValue;
+  fieldName?: string;
+  fieldValue?: any;
+  index?: number;
 }
 
 interface SohoColumnGroupSelected {
-  groupName: string;
-  groupValue: any;
+  // USe either index or groupName and groupValue;
```

Comment:
Use ...?

Reply:
I think its good. The code below is what shows up for me as green (new code)

Reply:
oh ok I didnt notice that